lib/edamam.rb

```diff
@@ -0,0 +1,192 @@
+module Edamam
+  APP_ID = ENV.fetch('EDAMAM_APP_ID')
+  APP_KEY = ENV.fetch('EDAMAM_APP_KEY')
+  MEASUREMENT_URI_PREFIX = 'http://www.edamam.com/ontologies/edamam.owl#Measure_'.freeze
+  NUTRITION_DATA_URL = 'https://api.edamam.com/api/food-database/nutrients'.freeze
+  UPC_LOOKUP_URL = 'https://api.edamam.com/api/food-database/parser'.freeze
+
+  class << self
+    def get_grams_per_measurement_unit(food_id:, measurement_name:)
+      data = send_request(
+        body: {
+          ingredients: [
+            {
+              quantity: 1,
+              measureURI: "#{MEASUREMENT_URI_PREFIX}#{measurement_name.parameterize.underscore}",
```

Comment:
Let's add a `measurement_uri` method that takes in the name and returns the full encoded URL.